lib/serverspec/type/hadoop_config.rb

```diff
@@ -0,0 +1,27 @@
+module Serverspec::Type
+  class HadoopConfig < Base
+    def initialize(name=nil)
```

Comment:
Method signature of `initialize` of `Serverspec::Type::Base` had changed recently.
https://github.com/mizzy/serverspec/commit/e34b78e9250a1f52eeb01b0954eeb71d3a14bbf6

So please change this line like below.

```ruby
def initialize(name=nil, options={})
```